src/uwtools/tests/test_cli.py

```diff
@@ -69,6 +71,31 @@ def test__add_subparser_fv3(subparsers):
     ]
 
 
+def test__add_subparser_rocoto(subparsers):
+    cli._add_subparser_rocoto(subparsers)
+    assert subparsers.choices[STR.rocoto]
+
+
+def test__add_subparser_rocoto_realize(subparsers):
+    cli._add_subparser_rocoto_realize(subparsers)
+    assert subparsers.choices[STR.realize]
+
+
+def test__add_subparser_rocoto_validate(subparsers):
+    cli._add_subparser_rocoto_validate(subparsers)
+    assert subparsers.choices[STR.validate]
```

Comment:
There were a handful of missing (though fairly trivial) `rocoto`-related CLI tests. Added here.